### ISSUE TITLE
Crash Report - Testnet 3.3 - CrisF#3405 

### ISSUE THREAD

**Issue by Cris-F** [CodaProtocol/coda#5984]
Snarkworker and blockproducer running for 4+ hours.
Just observing at the time of the crash.
Many prints such as:
[coda_crash_report_2020-09-17_08-31-49.154225.tar.gz](https://github.com/CodaProtocol/coda/files/5237727/coda_crash_report_2020-09-17_08-31-49.154225.tar.gz)
 
```
2020-09-17 08:54:26 UTC [Info] validating a new pubsub message ...
2020-09-17 08:54:27 UTC [Info] why u fail to validate :(
2020-09-17 08:54:27 UTC [Info] why u fail to validate :(
2020-09-17 08:54:30 UTC [Info] starting refreshing cpl 0 with key CIQAAAMVU4AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA (routing table size was 0)
2020-09-17 08:54:30 UTC [Info] failed when refreshing routing table
``` ```
2020-09-17 08:57:26 UTC [Info] would have validated but it's from us!
2020-09-17 08:57:26 UTC [Info] validated!
2020-09-17 08:57:26 UTC [Info] why u fail to validate :(
2020-09-17 08:57:26 UTC [Info] validated!
```
Crash report attached.
[coda_crash_report_2020-09-17_08-31-49.154225.tar.gz](https://github.com/CodaProtocol/coda/files/5237736/coda_crash_report_2020-09-17_08-31-49.154225.tar.gz)

**Comment by Cris-F** [comment]
Prints as above continue upon restart.

**Comment by emberian** [comment]
One of the snark worker errors that should be fixed by #6036
